backend/app/services/chat/trigger/group_chat.py

```
@@ -81,41 +85,99 @@ async def notify_group_members_task_updated(
+        spec = task_json.get("spec", {})
+        linked_group = spec.get("linked_group")
+
+        logger.info(
+            f"[notify_group_members_task_updated] task_id={task.id}, linked_group={linked_group}, spec_keys={list(spec.keys())}"
         )
 
+        if linked_group:
+            # For linked group chats, get members from the linked Namespace (group)
+            namespace = (
+                db.query(Namespace)
+                .filter(Namespace.name == linked_group, Namespace.is_active == True)
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)